front_end/entrypoint_template.html

```diff
@@ -14,7 +14,7 @@
     }
   }
 </style>
-<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
+<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' 'unsafe-eval' 'unsafe-inline' https://chrome-devtools-frontend.appspot.com">
```

Comment:
I think its better to move this logic to `scripts/build/generate_html_entrypoint.js`, to make in `rn_fusebox`-only same as I did in https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/23/files#diff-f40b55ac87b2a03f4ad7aa51a7b6deabc30cf478316ed3501b47b7ec8a85995d